### PR TITLE
edit workflow.mobile_friendly with workflow.configuration.swipe_enabled

### PR DESCRIPTION
**Pull request by mcbouslog**
Staging branch URL: https://workflow-mobile-friendly.pfe-preview.zooniverse.org/

Related to https://github.com/zooniverse/mobile/issues/151.

Adds functionality so workflow editor mobile section edits `workflow.mobile_friendly` boolean similarly to `workflow.configuration.swipe_enabled`. This will allow API request in mobile app for projects' workflows to be filtered by `mobile_friendly`.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/pages/lab/mobile/mobile-section-container.fixtures.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ function workflow(extraData = {}) {
     },
     configuration: {
       swipe_enabled: true
-    }
+    },
+    mobile_friendly: true
```
Comment (wgranger) [inline app/pages/lab/mobile/mobile-section-container.fixtures.js]: With the rebase, this line is now a repeat of line 51 above.

**Comment by wgranger** [comment]
Thanks for the PR. This strengthens the code a bit for the enabled variable and adds a bit more to testing. This should be good to merge after the comment above is addressed.

**Comment by mcbouslog** [comment]
removed duplicate attribute, amended most recent commit

**Comment by wgranger** [comment]
Cool. Looks good.

**Pull request merged**